src/main/java/technology/semi/weaviate/client/WeaviateAuthClient.java

```diff
@@ -0,0 +1,23 @@
+package technology.semi.weaviate.client;
```

Comment:
What happens if a user provides an authentication header and WeaviateAuthClient? In the python client, we emit a warning and ignore the header, eg WeaviateAuthClient has precedent

Reply:
The `Authorization` header is added dynamically to headers if `WeaviateAuthClient` was created, that operation overrides previous value

Reply:
Should we add a warning (+test) in case both are present?

Reply:
I have a test that checks which value will be saved when multiple inserts with the same key occur, I think that we can skip the warning